lib/rubocop/cop/rspec/factory_bot/static_attribute_defined_dynamically.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Prefer declaring static attribute values without a block.
+        #
+        # @see DynamicAttributeDefinedStatically
+        #
+        # @example
+        #   # bad
+        #   kind { :static }
+        #
+        #   # good
+        #   kind :static
+        #
+        #   # bad
+        #   comments_count { 0 }
+        #
+        #   # good
+        #   comments_count 0
+        #
+        #   # bad
+        #   type { User::MAGIC }
+        #
+        #   # good
+        #   type User::MAGIC
+        class StaticAttributeDefinedDynamically < Cop
+          MSG = 'Do not use a block to set a static value ' \
+                'to an attribute.'.freeze
+
+          def_node_matcher :block_value_matcher, <<-PATTERN
+            (block (send nil? _) _ $...)
+          PATTERN
+
+          def_node_search :factory_attributes, <<-PATTERN
+            (block (send nil? { :factory :trait } ...) _ { (begin $...) $(send ...) $(block ...) } )
+          PATTERN
+
+          def on_block(node)
+            factory_attributes(node).to_a.flatten.each do |attribute|
+              values = block_value_matcher(attribute)
+              next if values.to_a.none? { |v| static?(v) }
+              add_offense(attribute, location: :expression)
+            end
+          end
+
+          def autocorrect(node)
+            lambda do |corrector|
+              corrector.replace(
+                node.loc.expression,
+                "#{node.descendants.first.source} #{node.body.source}"
```

Comment:
I guess `node.descendants.first` is actually `node.send_node`

Reply:
Sounds good 👍